### ISSUE TITLE
Parameter mistake in CE loss

### ISSUE THREAD

**Issue by LinfengGao** [huggingface/jat#165]
https://github.com/huggingface/jat/blob/010477208a2c647c7b02b0a6dd691db142cabb7a/jat/modeling_jat.py#L85
Should here be `end_dim=1` according to the comment? I'm a beginner and not sure about that.

**Comment by qgallouedec** [comment]
Thanks for reporting @LinfengGao, good catch!

Hopefully, this is not an issue for 2 reasons:
1. `logits` and `label` are eventually both reshaped to `B*L*X, C` in loss computation:

https://github.com/huggingface/jat/blob/010477208a2c647c7b02b0a6dd691db142cabb7a/jat/modeling_jat.py#L90

(Which makes me realize that the line you're referring is useless, I'll remove it)

2. This section of the code is never used anyway, because `compute_ce_loss` is never called without a mask.

**Comment by LinfengGao** [comment]
Thanks for your explanation!
